ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -106,6 +106,13 @@ def _prepareCommand( self ):
       if not seHost:
         return S_ERROR( 'No seHost for %s' % elementName )
       elementName = seHost
+      
+    elif elementType == 'FTS':
+    	gocdbServiceType = "FTS"
+    	try:
+    		elementName	= getFTSServers("FTS3")[ 'Value' ][0]
+    	except:
+    		return S_ERROR( 'No FTS3 server specified in dirac.cfg (see Resources/FTSEndpoints)' )
```

Comment:
I simply thought for now it was enough to deal with a unique centralized server. If you can explain better to me how you want this to be handled, I'll do as you say.